blocks/large-manual-promo-block/features/large-manual-promo/default.jsx

```diff
@@ -30,12 +30,12 @@ const OverlineHeader = styled.h2`
   text-decoration: none;
 `;
 
-const LargeManualPromo = ({ customFields }) => {
+const LargeManualPromoItem = ({ customFields }) => {
   const { arcSite } = useFusionContext();
   const textClass = customFields.showImage ? 'col-sm-12 col-md-xl-6 flex-col' : 'col-sm-xl-12 flex-col';
 
   const resizedImageOptions = useContent({
-    source: 'resize-image-api',
+    source: customFields.lazyLoad ? 'resize-image-api-client' : 'resize-image-api',
```

Comment:
we should be using client side resizer just for isAdmin right? trying to remember this conversation

Reply:
No.. This is actually a better experience for editors as the client side content source will run in Admin as well meaning the editor user does not have to refresh the preview panel to see image changes

Reply:
ok I must've misread this. we want client-side resiszing for manual promos for admins regardless of opting into lazyLoad